packages/gatsby-source-shopify/src/type-builders/location-type.ts

```diff
@@ -38,6 +37,11 @@ export function locationTypeBuilder(prefix: string): string {
         variant: ${prefix}ProductVariantConnection!
       }
 
+      type ${prefix}InventoryItemMeasurement {
+        id: ID!
```

Comment:
We might not need `ID` but measurement is not returning from the api at the moment